data/human/human missions.txt

```diff
@@ -3584,3 +3585,42 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Courier 5"
+	name "Yottrite samples to <planet>"
+	deadline
+	description "Deliver a few Yottrite samples to <destination> by <date>. Payment is <payment>."
+	minor
+	cargo "Yottrite samples" 10
+	to offer
+		random < 5
+	source
+		government "Quarg"
+	destination
+		distance 2 13
+		government "Republic"
+		attributes "rich"
+	on offer
+		conversation
+			`A man waves at you from afar, trying to get your attention."`
+			choice
+				`	(Find out what he wants.)`
+				`	(Ignore him.)`
+					decline
+			`	"My name is Jason ," he says. "I've gotten my hands on a few Yottrite samples that I bought from a Quarg. We need you to send them to a research center."`
```

Comment:
```suggestion
			`	"My name is Jason," he says. "I've gotten my hands on a few Yottrite samples that I bought from a Quarg. We need you to send them to a research center."`
```
_Is_ there casual trade like this between humans and Quarg? Dunno. It feels odd to me, though. Anyone else have thoughts?